dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -111,14 +117,22 @@ internal class OkHttpUploader : Uploader {
         }
     }
 
+    private fun RequestBody.sizeInMb(): Long {
+        val megabyte = 1024L * 1024L
```

Comment:
Actually here I need to make a round to the biggest long otherwise we can have sizes of `50.x` that will be considered valid.